feature_server/commands.py

```diff
@@ -524,6 +560,11 @@ def god(connection, value = None):
         connection.god_build = connection.god
     else:
         connection.god_build = False
+    return 'You have entered god mode.'
+
+@admin
+def god(connection, value = None):
+    godsilent(connection, value)
```

Comment:
I don't undertand why is `godsilent` called from within `god`, can anhone please explain?